openfisca_nouvelle_caledonie/parameters/prelevements_obligatoires/impot_revenu/revenus_imposables/tspr/abattement_pension/plafond.yaml

```
@@ -1,11 +1,18 @@
 description: Plafond de l'abattement sur les pensions
 values:
-  2024-01-01:
-    value: 550_000
+  2021-09-25:
+    value: 500000
+  2023-11-21:
+    value: 550000
 metadata:
-  short_label: Plafond
+  short_label: Maxime BOLLENGIER
```

```suggestion
  short_label: Plafond
```

Cette entrée est un label raccourci de la description qui est censé tenir compte du contexte.
Voir cette doc https://baremes-ipp-doc.tax-benefit.org/